KeenClient/KeenClient.m

```diff
@@ -412,6 +446,16 @@ - (void)locationManager:(CLLocationManager *)manager
     }
 }
 
+-(void)locationManager:(CLLocationManager *)manager didChangeAuthorizationStatus:(CLAuthorizationStatus)status {
+    if ([KeenClient isLocationAuthorized:status]) {
+        [self startMonitoringLocation];
+    }
+}
+-(void)locationManager:(CLLocationManager *)manager didFailWithError:(NSError *)error
+{
+  
```

Comment:
How would you suggest I do that? Just with a log statement or are you suggesting storing it as an event?